roles/edpm_update/molecule/default/verify.yml

```diff
@@ -0,0 +1,54 @@
+- name: Verify OVS update
+  hosts: all
+  vars:
+    test_helper_dir: "../../../../molecule/test-helpers"
+    test_vars: "{{molecule_yml['provisioner']['inventory']['host_vars']['test_vars']}}"
+  tasks:
+    - name: Ensure openvswitch systemd services are in appropriate states after the update
+      ansible.builtin.include_tasks: "{{test_helper_dir}}/verify_systemd_unit.yaml"
+      loop:
+        # "osp_service": false, disable check for unit files in /etc/systemd/system
+        # OVS systemd unit files are located in /usr/lib/systemd/system
+        - { "name": "openvswitch.service",
+            "osp_service": false,
+            "enabled": "enabled",
+            "active":  ["active"] }
+        - { "name": "ovs-vswitchd.service",
+            "osp_service": false,
+            "enabled": "static",
+            "active":  ["active"] }
+        - { "name": "ovsdb-server.service",
+            "osp_service": false,
+            "enabled": "static",
+            "active": ["active"] }
+
+    - name: Read stored openvswitch service pids
+      ansible.builtin.slurp:
+        src: "{{test_vars['working_directory']}}/{{test_vars['ovs_pid_file']}}"
+      register: pids_before_update
+
+    - name: Collect openvswitch service pids after update
+      ansible.builtin.shell:
+        pgrep "ovs(db-server|-vswitch)"
+      register: pids_after_update
+
+    - name: Assert openvswitch service pids did not change after the update
+      ansible.builtin.assert:
+        that: pids_before_update.content | b64decode | split('\n') ==
+              pids_after_update.stdout_lines
```

Comment:
is it guaranteed that the order of lines in each `pgrep` run is the same? (I don't see it documented in the man page for the tool.)